examples/anthropic-tool/README.md

```diff
@@ -13,34 +13,34 @@ This example demonstrates how to use Anthropic's Claude 3 models with native too
 ## Prerequisites
 
 - Node.js 16+
-- An Anthropic API key
+- A Google API key
 
 ## Setup
 
-1. Make sure you have an Anthropic API key. If you don't have one, you can get it from the [Anthropic Console](https://console.anthropic.com/)
+1. Make sure you have a Google API key. If you don't have one, you can get it from the [Google AI Studio](https://makersuite.google.com/)
 
 2. Set up your environment variables:
    - Create a `.env` file in the root directory or set the environment variable directly
-   - Add your Anthropic API key: `ANTHROPIC_API_KEY=your_key_here`
+   - Add your Google API key: `GOOGLE_API_KEY=your_key_here`
 
 ## Running the Example
 
 ```bash
 # From the project root
 npm run build
-npm run example:anthropic
+npm run example:anthropic  # Note: The script name remains the same for compatibility
 ```
 
 ## How It Works
 
-This example showcases native tool calling with Claude 3 models:
+This example showcases tool calling with Google Gemini models:
 
-1. Uses Claude 3 Sonnet which fully supports tool calling
+1. Uses Gemini 2.0 Flash which supports tool calling
```

Comment:
2.5

Reply:
True! Just testing with 2.0 cuz 2.5 seems to be slow. Assumed cuz its from any rate limit as its still experimental